packages/babel-preset-env/src/index.ts

```diff
@@ -449,3 +438,55 @@ option \`forceAllTransforms: true\` instead.
 
   return { plugins };
 });
+
+// TODO(Babel 8): This is only here for backward compatibility. Remove it.
+export { getModulesPluginNamesBackwardCompat as getModulesPluginNames };
+const getModulesPluginNamesBackwardCompat = ({
```

Comment:
This function is not changed, I just moved it down there and exported it for backward compat. I'll open a PR removing it for Babel 8.